### ISSUE TITLE
ToolTip AutoHide

### ISSUE THREAD

**Issue by FriedrichF** [qooxdoo/qooxdoo#9358]
The AutoHide Property in ToolTip is useless. It would be nice to add this into the ToolTip Manager.
```
__onHideInterval : function(e)
    {
      var current = this.getCurrent();
	  if(!current.getAutoHide())
		return;
      if (current && !current.isDisposed()) {
        current.exclude();
      }

      this.__hideTimer.stop();
      this.resetCurrent();
    },
```

```
__onPointerOutRoot : function(e)
    {
      var target = qx.ui.core.Widget.getWidgetByElement(e.getTarget());
      if (!target) {
        return;
      }

      var related = qx.ui.core.Widget.getWidgetByElement(e.getRelatedTarget());
      if (!related && e.getPointerType() == "mouse") {
        return;
      }

      var tooltip = this.getCurrent();

      // If there was a tooltip and
      // - the destination target is the current tooltip
      //   or
      // - the current tooltip contains the destination target
      if (tooltip && (related == tooltip || qx.ui.core.Widget.contains(tooltip, related))) {
        return;
      }

      // If the destination target exists and the target contains it
      if (related && target && qx.ui.core.Widget.contains(target, related)) {
        return;
      }
	  
	  if(!tooltip.getAutoHide())
		return;

      // If there was a tooltip and there is no new one
      if (tooltip && !related) {
        this.setCurrent(null);
      } else {
        this.resetCurrent();
      }
    },
```

```
__onFocusOutRoot : function(e)
    {
      var target = qx.ui.core.Widget.getWidgetByElement(e.getTarget());
      if (!target) {
        return;
      }

      var tooltip = this.getCurrent();
	  
	  if(!tooltip.getAutoHide())
		return;

      // Only set to null if blured widget is the
      // one which has created the current tooltip
      if (tooltip && tooltip == target.getToolTip()) {
        this.setCurrent(null);
      }
    }
```

**Comment by johnspackman** [comment]
Please can you submit this as a pull request?

**Comment by level420** [comment]
Fixed via #9359
